spec/features/case_management/tax_return_status_change_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "rails_helper"
+
+RSpec.feature "Change tax return status on a client" do
+  context "As a beta tester" do
+    let(:vita_partner) { create :vita_partner }
+    let(:logged_in_user) { create :user, vita_partner: vita_partner }
+    let(:client) { create :client, vita_partner: vita_partner }
+    let!(:intake) { create :intake, client: client }
+    let!(:tax_return) { create :tax_return, year: 2019, client: client, status: "intake_in_progress" }
+
+    before do
+      login_as logged_in_user
+    end
+
+    scenario "logged in user can change a status on a tax return" do
+      visit case_management_client_path(id: client.id)
+      expect(page).to have_select("tax_return_status", selected: "In progress")
+
+      within "#tax-return-#{tax_return.id}" do
+        select "Open", from: "tax_return_status"
+
+        click_on "Update"
+      end
+
+      expect(current_path).to eq(edit_status_case_management_client_tax_return_path(id: tax_return.id, client_id: tax_return.client.id))
+      expect(page).to have_select("tax_return_status", selected: "Open")
+
+      click_on "Save"
+      expect(current_path).to eq(case_management_client_messages_path(client_id: tax_return.client.id))
+      expect(page).to have_select("tax_return_status", selected: "Open")
```

Comment:
This appears to keep the same status over the course of editing it, namely `"Open"`. I'd like to see a test that validates that it saves properly.

Reply:
you change the status on the first page, and then it takes you to this other page to persist the change, based on the pivotal tracker ticket. that's why its the same status when you get to the page after changing it on the previous page. but an additional test would make sense.

Reply:
I didn't see the status change on the earlier lines! Makes sense now.